local-cluster/local-cluster.cabal

```diff
@@ -84,6 +82,7 @@ library
     , hspec-expectations
     , http-conduit
     , hydra-prelude
+    , hydra-node
```

Comment:
As we have this here anyway, I guess I'd prefer the Hydra.Network.Ports to be in `hydra-node` instead of the `hydra-prelude`. May I move it or WDYT?